test/statetest/statetest_loader.cpp

```diff
@@ -251,6 +251,9 @@ static void from_json_tx_common(const json::json& j, state::Transaction& o)
 {
     o.sender = from_json<evmc::address>(j.at("sender"));
 
+    if (const auto nonce_it = j.find("nonce"); nonce_it != j.end())
```

Comment:
Isn't the nonce always expected there?

Reply:
Our `load_minimal_test` test does not define nonce. So we can add that it's mandatory or we load it only when it's defined.